backend/Dockerfile

```diff
@@ -1,20 +1,24 @@
-FROM python:3.7.4-alpine
+FROM python:3.7-slim
 
-RUN mkdir /JDISCTF
-WORKDIR /JDISCTF
+WORKDIR /flaggr
 
-COPY requirements.txt /JDISCTF
+COPY requirements.txt .
+RUN pip install gunicorn==19.10.0 --no-cache-dir
+RUN pip install -r requirements.txt --no-cache-dir && \
+ find /usr/local \
```

Comment:
What does this do?

Reply:
It removes all compiled python stuff (.pyc) and tests from the dependencies